vispy/visuals/graphs/layouts/circular.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+"""
+Circular Layout
+===============
+
+This module contains several graph layouts which rely heavily on circles.
+"""
+
+import numpy as np
+
+from ..util import _straight_line_vertices
+
+
+def circular(adjacency_mat, directed=False):
+    """Places all nodes on a single circle.
+
+    Parameters
+    ----------
+    adjacency_mat : matrix or sparse
+        The graph adjacency matrix
+    directed : bool
+        Whether the graph is directed. If this is True, is will also
+        generate the vertices for arrows, which can be passed to an
+        ArrowVisual.
+
+    Yields
+    ------
+    (node_vertices, line_vertices, arrow_vertices) : tuple
+        Yields the node and line vertices in a tuple. This layout only yields a
+        single time, and has no builtin animation
+    """
+
+    num_nodes = adjacency_mat.shape[0]
+
+    t = np.arange(0, 2.0*np.pi, 2.0*np.pi/num_nodes, dtype=np.float32)
```

Comment:
this would be a good use for `t = np.linpsace(0, 2 * np.pi, num_nodes, endpt=False, dtype=np.float32)`